src/test/java/org/fundacionjala/coding/jimmy/Kata5Test.java

```diff
@@ -0,0 +1,92 @@
+package org.fundacionjala.coding.jimmy;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Class for performing unit tests for makeComplement method.
+ */
+public class Kata5Test {
+
+    private Kata5 kata5;
+
+    /**
+     * Initializes kata5.
+     */
+    @Before
+    public void initialize() {
+        kata5 = new Kata5();
+    }
+
+    /**
+     * Unit test for testing makeComplement method.
+     */
+    @Test
+    public void test01() {
+        final String dnaString = "AAAA";
+        final String expected = "TTTT";
+        assertEquals(expected, kata5.makeComplement(dnaString));
+    }
+
+    /**
+     * Unit test for testing makeComplement method.
+     */
+    @Test
+    public void test02() {
+        final String dnaString = "ATTGC";
+        final String expected = "TAACG";
+        assertEquals(expected, kata5.makeComplement(dnaString));
+    }
+
+    /**
+     * Unit test for testing makeComplement method.
+     */
+    @Test
+    public void test03() {
+        final String dnaString = "GTAT";
+        final String expected = "CATA";
+        assertEquals(expected, kata5.makeComplement(dnaString));
+    }
+
+    /**
+     * Unit test for testing makeComplement method.
+     */
+    @Test
+    public void test04() {
+        final String dnaString = "AAGG";
+        final String expected = "TTCC";
+        assertEquals(expected, kata5.makeComplement(dnaString));
+    }
+
+    /**
+     * Unit test for testing makeComplement method.
+     */
+    @Test
+    public void test05() {
+        final String dnaString = "CGCG";
+        final String expected = "GCGC";
+        assertEquals(expected, kata5.makeComplement(dnaString));
+    }
+
+    /**
+     * Unit test for testing makeComplement method.
+     */
+    @Test
+    public void test06() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Update this method so that its implementation is not identical to "test02" on line 37. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4144)